compiler/luci/pass/src/ResolveCustomOpAddPass.cpp

```diff
@@ -25,30 +25,49 @@
 namespace
 {
 
+/// @brief Returns the number of BroadcastTo(Custom) node among cop's inputs
```

Comment:
`BroadcastTo(Custom)` is strange.
- As this is `CircleCustom` Op, Custom("BroadcastTo") would be better

Reply:
@seanshpark 

Thank you for your comments !
I reflected these in this PR, 
so Could you review the code again? :)